### PR TITLE
Add a warning for a correct restore

### PR DESCRIPTION
**Pull request by nickvergessen**
@MorrisJobke

### CODE REVIEW

File: admin_manual/maintenance/restore.rst [hunk 1]
```diff
@@ -30,6 +30,38 @@ install and data folder) to your Nextcloud environment. You could use this comma
 Restore Database
 ----------------
 
+Clean Database before restoring
```
Comment (ChristophWurst) [inline admin_manual/maintenance/restore.rst]: From looking at the other headings I assume we should use caps for all words: "Clean Database Before Restoring"

File: admin_manual/maintenance/restore.rst [hunk 2]
```diff
@@ -30,6 +30,38 @@ install and data folder) to your Nextcloud environment. You could use this comma
...
+
+MySQL
+^^^^^
+
+MySQL is the recommended database engine. To restore MySQL::
+
+   mysql -h [server] -u [username] -p[password] -e "DROP DATABASE nextcloud"
+   mysql -h [server] -u [username] -p[password] -e "CREATE DATABASE nextcloud"
+
+If you use UTF8 with multibyte support (e.g. for emoijs in filenames), use::
```
Comment (ChristophWurst) [inline admin_manual/maintenance/restore.rst]: is it possible to restore a non-multibyte backup into a multibyte database?

Comment (nickvergessen) [inline admin_manual/maintenance/restore.rst]: Jup

File: admin_manual/maintenance/restore.rst (hunk 1 above)
Comment (nickvergessen) [inline admin_manual/maintenance/restore.rst]: Fixed

**Pull request merged**